test/integration/files.js

```diff
@@ -12,6 +12,8 @@ const COZY_STACK_URL = process.env && process.env.COZY_STACK_URL || ''
 const COZY_STACK_VERSION = process.env && process.env.COZY_STACK_VERSION
 const COZY_STACK_TOKEN = process.env && process.env.COZY_STACK_TOKEN
 
+global.navigator = undefined
```

Comment:
Why do you need to instantiate `gobal.navigator` to `undefined`.
If you don't declare `global.navigator`, then `global.navigator` is `undefined`...